script.module.dateutil/NEWS

```diff
@@ -1,3 +1,100 @@
+Version 2.6.1
```

Comment:
Maybe rename this to changelog.txt?

Reply:
No that's how upstream named the file and I don't change those filenames. There is also no need that users can see the changelog.

Reply:
Yeap, ignore. I forgot this is a module.